packages/project-utils/__tests__/bundling/function/telemetryFunction.test.ts

```diff
@@ -1,5 +1,12 @@
-const path = require("path");
-const fs = require("fs");
+import * as fs from "fs";
+import { getProject } from "@webiny/cli/utils";
+import { getTelemetryFunctionDownloadPath } from "../../../bundling/function/utils";
+import { downloadTelemetryFunction } from "../../../bundling/function/telemetry";
+import { sleep } from "../../../testing/helpers/sleep";
+import * as path from "path";
+
+// This environment API key has been previously created via a test WCP account created at https://app.webiny.com. Consult internal documentation for more information on the used account.
+const VALID_API_KEY = "8c8d2096-934e-47b0-ab02-25feace28d48";
```

Comment:
This is not the API key I sent the other day:
https://webiny.slack.com/archives/D02ADBU9URW/p1647954900057159